Makefile

```diff
@@ -64,6 +64,8 @@ mypy:
 	grep ChannelID mypy-out.txt; [ $$? -eq 1 ]
 	grep BalanceProof mypy-out.txt; [ $$? -eq 1 ]
 	grep SendSecret mypy-out.txt; [ $$? -eq 1 ]
```

Comment:
just a note Paul. I think I am not opting for this kind of fixes. I am now taking each file and fixing it on its own so that we can soon just run mypy on the whole thing.

Reply:
The advantage of doing this is that it prevents regressions, but not all fixes are easily checkable by this.

Reply:
yeah what I mean is we planned to just finish this issue kind of soonish. We started with this method and kind of stopped. So I want to just do it now and be done with it, as we decided to take these 2 weeks for the "tech debt" stuff.

We want raiden to become like the services and just run mypy on the whole codebase asap. So I am planning to make many small PRs like this one or the one you approved from me a few mins ago.